### PR TITLE
(DOCSP-27544): Flutter MapTo for realm models

### PR DESCRIPTION
**Pull request by mongodben**
## Pull Request Info

Flutter MapTo for realm models

> **Note**
> 
> Intentionally not going into much detail in the description b/c i think each of these topics would take up a lot of space. Whereas here I just am going for a minimalist API description of this utility annotations so that users can decide what they do with it.

### Jira

- https://jira.mongodb.org/browse/DOCSP-27544

### Staged Changes

- [Define a Realm Object Schema (Flutter)](https://docs-mongodbcom-staging.corp.mongodb.com/realm/docsworker-xlarge/DOCSP-27544/sdk/flutter/realm-database/model-data/define-realm-object-schema/)

### Reminder Checklist

If your PR modifies the docs, you might need to also update some corresponding
pages. Check if completed or N/A.

- [x] Create Jira ticket for corresponding docs-app-services update(s), if any
- [x] Checked/updated Admin API
- [x] Checked/updated CLI reference

### Review Guidelines

[REVIEWING.md](https://github.com/mongodb/docs-realm/blob/master/REVIEWING.md)

### CODE REVIEW

**Review by desistefanova — COMMENTED** [review]
LGTM

File: source/sdk/flutter/realm-database/model-data/define-realm-object-schema.txt [hunk 1]
```diff
@@ -187,6 +187,26 @@ inverse relationships, and embedding objects within other objects.
 To learn more about how to define relationships in your Realm object schema,
 refer to :ref:`Relationships <flutter-client-relationships>`.
 
+.. _flutter-map-model-name:
+
+Map Realm Model to Different Name
+---------------------------------
+
+You can use the `MapTo <https://pub.dev/documentation/realm_common/latest/realm_common/MapTo-class.html>`__
+annotation to map a Realm object model to a different table name in Realm Database.
```
Comment (dacharyc) [inline source/sdk/flutter/realm-database/model-data/define-realm-object-schema.txt:196]: Nit: we don't really talk about tables generally in Realm Database, as we focus on Realm being object-driven. It stood out to me that we say "table" here. Could we say instead "different model name" or something along those lines?

**Review by dacharyc — APPROVED** [review]
Overall, LGTM - just a minor nit.

File: source/sdk/flutter/realm-database/model-data/define-realm-object-schema.txt (hunk 1 above)
Comment (mongodben) [reply source/sdk/flutter/realm-database/model-data/define-realm-object-schema.txt:196]:
```suggestion
annotation to map a Realm object model to a different model name in Realm Database.
```

**Pull request merged**